app/models/vm_synchronize.rb

```diff
@@ -18,7 +18,7 @@ def call_synchronize
 
     options[:categories] ||= Vm.default_scan_categories
     begin
-      host = Host.find(agent_id)
+      host = MiqServer.find(agent_id)
```

Comment:
@gtanzillo I thought Sync was part of the VmScan job. The only place I see this referenced is in: 
```
/Users/rpo/src/git/cfme/spec/lib/miq_automation_engine/miq_ae_service_spec.rb
```
but I'm not sure why. @gmcculloug any idea?

@jerryk55 I know you've been looking into the scan jobs, is this still used?